tracer/test/test-applications/debugger/Directory.Build.props

```diff
@@ -1,19 +1,19 @@
 <Project>
   <PropertyGroup>
-    <!-- only run .NET Framework tests on Windows -->
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net461;$(TargetFrameworks)</TargetFrameworks>
+    <RuntimeIdentifiers Condition="'$(OS)' == 'Windows_NT' AND '$(TargetFramework)' == 'net461'">win7-x86;win7-x64</RuntimeIdentifiers>
+  </PropertyGroup>
```

Comment:
This was updated recently to reduce the number of tested frameworks in PRs - you probably want to do something similar?

Reply:
Makes sense - we're only conditionally running these tests anyway, so I think that's fine.